packages/@react-aria/ssr/src/SSRProvider.tsx

```diff
@@ -79,6 +76,23 @@ export function SSRProvider(props: SSRProviderProps): JSX.Element {
   );
 }
 
+let warnedAboutSSRProvider = false;
+
+/**
+ * When using SSR with React Aria in React 16 or 17, applications must be wrapped in an SSRProvider.
+ * This ensures that auto generated ids are consistent between the client and server.
+ */
+export function SSRProvider(props: SSRProviderProps) {
+  if (typeof React['useId'] === 'function') {
+    if (process.env.NODE_ENV !== 'test' && !warnedAboutSSRProvider) {
```

Comment:
How likely are other projects to test with SSR in their unit tests do you think? I guess it'd hopefully be the same between their tests and browser, so if they see the warnings in the test they'd see it in the browser anyways too